bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,200 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="tapocontrol"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- ############################### BRIDGE ############################### -->
+	<bridge-type id="bridge">
+		<label>TapoControl Cloud-Login</label>
```

Comment:
The binding name is redundant and could therefore be removed.

Please check all.
```suggestion
		<label>Cloud Login</label>
```